mongodb-atlas-mixin/config.libsonnet

```diff
@@ -0,0 +1,20 @@
+{
+  _config+:: {
+    // sharding dashboard flag
+    enableShardingOverview: false,
```

Comment:
Just making sure that this is specced to be false by default?

Reply:
not really specced anywhere, but set it to false by default because as far as i can tell sharded clusters are not the default for atlas, you have to use their advanced config page to make a cluster sharded